README-zh.md

```diff
@@ -13,7 +13,14 @@ This repository contains the assets required to build the [Kubernetes website an
 我们非常高兴您想要参与贡献！
 
 <!--
-# Using this repository
+- [Contributing to the docs](#contributing-to-the-docs)
+- [Localization ReadMes](#localization-readmemds)
+-->
+- [为文档做贡献](#为文档做贡献)
```

Comment:
标题的锚点不要翻译，但是要在相应的中文标题后面加上 {#xxxx}
```suggestion
- [为文档做贡献](#contributing-to-the-docs)
```

Reply:
I know generally we should add ` {#anchor}` since non-ASCII character are ignored in anchor generation.

But it's `README-zh.md` for GitHub, there are unfortunately no such syntax to customize anchor ID.

Reply:
oops, my bad